mlflow/.env

```diff
@@ -0,0 +1,3 @@
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=password
+POSTGRES_DB=mlflow
```

Comment:
This file should not be added to the repository. Eventually, you can add this file with empty values to guide users of this repository on which variables should be provided.